src/components/side-by-side/side-by-side.tsx

```diff
@@ -2,14 +2,15 @@ import {h, createRef, Component} from 'preact';
 import * as styles from './side-by-side.scss';
 import {icons} from '../../icons';
 import {Button} from './../button';
-import {Animations} from './../../enums';
+import {Animations, StreamMode} from './../../enums';
 const {Icon} = KalturaPlayer.ui.components;
 const {connect} = KalturaPlayer.ui.redux;
 const {withText, Text} = KalturaPlayer.ui.preacti18n;
 
 const translates = () => {
   return {
-    expandScreen: <Text id="dualScreen.expand_screen">Expand Screen</Text>
+    expandPrimaryScreen: <Text id="dualScreen.expand_primary_screen">Expand primary screen</Text>,
```

Comment:
check `StreamMode` here and pass to component only one prop that should be rendered